主页翻译

inkydragon opened the pull request:
+ 尽力本地化，默认的文档链接均指向中文文档

inkydragon committed:
[metadata/lang code] follow mozilla's choice

inkydragon committed:
[metadata/ogp]

ref: https://ogp.me/

inkydragon committed:
[metadata/html] 本地化网站简介，优化 SEO

Note: 这里的 description 会作为搜索引擎显示的简介

ref: https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/meta

inkydragon committed:
[js] 关闭GA统计

可以使用本地化的统计插件。

使用 GA 则需要跨域统计设置
ref: https://support.google.com/analytics/answer/1034342?hl=zh-Hans

inkydragon committed:
[js] 换用国内 cdn

+ fonts.googleapis.com 目前看起来速度很快，并且暂无 cdn 替代

inkydragon committed:
删除冗余的注释

highlight.js 已经移动至 foot_highlight.html

inkydragon committed:
[navbar] 指向中文文档

inkydragon committed:
[简而言之] 删除多余的空格

inkydragon committed:
大标题

inkydragon committed:
[基本工具]

inkydragon committed:
[页脚]

- 删除了 youtube 频道（官方频道中仍有链接）

锚点失效
+ 下载：稳定版；LTS
+ 社区：调查、官方频道、活动

死链
+ /community/#julia_github_groups

inkydragon committed:
[blog, 必备工具]

inkydragon committed:
[赞助，star]

inkydragon committed:
指向中文文档

inkydragon committed:
[JuliaCon]

inkydragon committed:
[语言生态] 可视化、通用编程

inkydragon committed:
[软件包]

findmyway commented:
赞！

有一个稍微注意下，Google Analytics 别关了，换成跟 https://docs.juliacn.com/latest/ 这里一样的 ID，这个做数据分析的时候还是有点用的。

inkydragon committed:
[关注我们]

+ SVG 来自维基百科；知乎 logo 略作修改
1. https://zh.wikipedia.org/wiki/File:Tencent_QQ.svg
2. https://zh.wikipedia.org/wiki/File:Zhihu_logo.svg

inkydragon committed:
更新 GA 设置

inkydragon committed:
[语言生态] 翻译完毕

inkydragon committed:
[语言生态] 删除不会显示的 div

inkydragon committed:
文档链接指向中文文档

inkydragon committed:
[infra/] 删除重复的文件

这些文件目前放在 `_assets/infra/` 中

findmyway commented:
话说，首页那个Youtube的链接可以换成B站的么？
